{
 "bad": false,
 "new_version": "2.7.3",
 "new_version_attempts": {
  "1.0.10183": 1,
  "1.0.10200": 1,
  "1.0.10395": 1,
  "1.0.10409": 1,
  "1.0.11104": 1,
  "1.0.11222": 1,
  "1.0.11279": 1,
  "1.0.12198.1": 1,
  "1.0.12260.1": 0,
  "1.0.12392.1": 1,
  "1.0.13822.1": 1,
  "1.0.13860.4": 1,
  "1.0.3698": 1,
  "1.0.3771": 1,
  "1.0.3826": 1,
  "1.0.3864": 1,
  "1.0.3899": 1,
  "1.0.3951": 1,
  "1.0.3977": 1,
  "1.0.4053": 1,
  "1.0.4062": 1,
  "1.0.4111": 1,
  "1.0.4154": 1,
  "1.0.4155": 1,
  "1.0.4241": 1,
  "1.0.4312": 1,
  "1.0.4361": 1,
  "1.0.4427": 1,
  "1.0.4479": 1,
  "1.0.4521": 1,
  "1.0.6712": 1,
  "1.0.6812": 1,
  "1.0.6909": 1,
  "1.0.7041": 1,
  "1.0.7152": 1,
  "1.0.7181": 1,
  "1.0.7423": 1,
  "1.0.7683": 1,
  "2.0.17791.2": 3,
  "2.2.1": 3,
  "2.2.2": 1,
  "2.2.3": 1,
  "2.3.1": 7,
  "2.4.3": 5,
  "2.5.10": 9,
  "2.5.11": 12,
  "2.5.4": 46,
  "2.5.5": 4,
  "2.5.6": 4,
  "2.5.7": 14,
  "2.6.2": 5,
  "2.6.3": 15,
  "2.7.3": 13
 },
 "new_version_errors": {
  "2.0.17791.2": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/11946261354\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name intel-graphics-compiler --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.0.17791.2' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\"",
  "2.2.1": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12079797316\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name intel-graphics-compiler --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.2.1' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\"",
  "2.2.2": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12143029227\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name intel-graphics-compiler --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.2.2' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\"",
  "2.2.3": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12179213199\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name intel-graphics-compiler --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.2.3' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\"",
  "2.3.1": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12351538289\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name intel-graphics-compiler --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.3.1' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\"",
  "2.4.3": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12430941882\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name intel-graphics-compiler --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.4.3' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\"",
  "2.5.10": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12825362551\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 254, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.5.10' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.5.11": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12869557149\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 254, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.5.11' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.5.4": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12637784728\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 218, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.5.4' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.5.5": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12671318973\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 218, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.5.5' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.5.6": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12682905640\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 218, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.5.6' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.5.7": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12754578670\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 218, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.5.7' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.6.2": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12880902610\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 254, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.6.2' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.6.3": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/12961367738\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 254, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.6.3' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n",
  "2.7.3": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/13001880949\">bot CI job</a>): main:\nTraceback (most recent call last):\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 119, in _run_bot_task\n    data = func(attrs=attrs, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/container_cli.py\", line 225, in _migrate_feedstock\n    data = run_migration_local(\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migration_runner.py\", line 251, in run_migration_local\n    data[\"migrate_return_value\"] = migrator.migrate(\n                                   ^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/migrators/version.py\", line 254, in migrate\n    raise VersionMigrationError(\nconda_forge_tick.migrators.version.VersionMigrationError: The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '2.7.3' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://github.com/intel/intel-graphics-compiler/archive/igc-{{ version }}.tar.gz'\n\n"
 }
}